### PR TITLE
Move run.sh shebang (#!/bin/bash) to top

### PR DESCRIPTION
**Pull request by NimJay**
### Background/Problem

- This fixes [#441](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/issues/441).
- If you build and run `/guestbook/redis-follower/Dockerfile`, the following error will be output:
```
/run.sh: 18: /run.sh: [[: not found
...
```

### Solution
* According the ["Google Copyright Notice Usage" doc](https://g3doc.corp.google.com/company/teams/opensource/copyright.md#build-shell-scripts-sawzall) (Google-internal doc), the shebang can be placed before the license header.
* So I have placed the shebang at the top of the file.

### Manually Tested
- I have built and run the `/guestbook/redis-follower` locally.
- I no longer see the error (`/run.sh: 18: /run.sh: [[: not found`).

### CODE REVIEW

File: guestbook/redis-follower/run.sh [hunk 1]
```diff
@@ -14,12 +15,11 @@
...
 
 if [[ ${GET_HOSTS_FROM:-dns} == "env" ]]; then
   redis-server --replicaof ${REDIS_LEADER_SERVICE_HOST} 6379
 else
   redis-server --replicaof redis-leader 6379
 fi
 
-# [END gke_redis_follower_run]
\ No newline at end of file
+# [END gke_redis_follower_run]
```
Comment (NimJay) [inline guestbook/redis-follower/run.sh:25]: This new line at the end of the file is technically unrelated to the problem (and out of scope), so let me know if I should undo it.

**Review by bourgeoisor — APPROVED** [review]
